app/controllers/api/v1/trails_controller.rb

```diff
@@ -1,12 +1,12 @@
 class Api::V1::TrailsController < ApiController
 
     def index
-
+        
         render json: Trail.all
     end
     
     def show
-
+        
```

Comment:
we should be reviewing each file we are going to add for a commit to ensure we do want all the changes - we can remove these empty lines with blank spaces!